packages/inter-protocol/src/feeDistributor.js

```diff
@@ -212,6 +212,7 @@ export const makeFeeDistributor = (feeIssuer, terms) => {
             return;
           }
           const payment = await E(collector).collectFees();
+          assert(payment, 'Missing payment from collector');
```

Comment:
that's a random drive-by correction. What justifies including it with this PR? Does it deserve a test?

Reply:
> What justifies including it with this PR? 

IIRC it's because I ran into an error that this would have helped me diagnose. So error detection.

> Does it deserve a test?

No because it's used only for `distributeFees()` which requires a real Payment.